### PR TITLE
Better levels

### PR DESCRIPTION
**Pull request by Williangalvani**
Improves loading external levels

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,3 +43,15 @@ If there is not SITL instance attached, these keys can be used to control the RO
     Before opening Bluesim (it launches its own SITL instance), run:
 
  `sim_vehicle.py -j6 -L RATBeach --frame JSON --out=udpout:0.0.0.0:14550`
+
+# External Levels
+
+External levels can be loaded by placing a .pck file in the "levels" folder at the same level as the simulator executable (the folder is created at first run if it does not exist).
```
Comment (patrickelectric) [inline README.md:49]: That's not happening, the folder does not exist after the simulator runs for the first time, for test, I just exported a linux binary and it's failing to create the levels folder.

File: menu/menu.tscn [hunk 2]
```diff
@@ -7,14 +7,45 @@
...
+	func list_available_external_levels():
+		var dir = Directory.new()
+		dir.open(\"res://levels\")
+		dir.list_dir_begin()
+		var found = []
+	
+		# list .pck files
+		while true:
+			var file = dir.get_next()
+			if file == \"\":
```
Comment (patrickelectric) [inline menu/menu.tscn:26]: file.empty() ?

**Review by patrickelectric — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Williangalvani) [reply README.md:49]: can you try now?

**Review by patrickelectric — APPROVED** [review]
working fine!

**Pull request merged**